lib/concurrent-ruby/concurrent/utility/processor_counter.rb

```diff
@@ -187,4 +212,12 @@ def self.available_processor_count
   def self.cpu_quota
     processor_counter.cpu_quota
   end
+
+  # The CPU shares requested by the process. For performance reasons the calculated
+  # value will be memoized on the first call.
+  #
+  # @return [Float, nil] CPU shares requested by the process, or nil if not set
+  def self.cpu_requests
```

Comment:
I wonder what's the best name here.
`cpu_shares` might be better since it reads from `/sys/fs/cgroup/cpu.weight` or `/sys/fs/cgroup/cpu/cpu.shares`.
And we already have `cpu_quota` which reads from `cpu.cfs_quota_us`/`cpu.max`.

`requests` sounds like a Kubernetes-specific term, notably Docker calls it shares in https://docs.docker.com/config/containers/resource_constraints/#cpu

So could you rename to `cpu_shares`?

Reply:
@eregon Thank you for suggestion. I've made changes based on comment.